lib/school.rb

```diff
@@ -0,0 +1,31 @@
+require 'time'
+
+class School
+ attr_reader :hours_in_school_day,
+             :student_names,
+             :start_time
+
+  def initialize(hours_in_school_day, start_time)
```

Comment:
This does not match the specification. Start time should come first, followed by hours in school day.

Reply:
Hi Mike, 

Thank you for pointing that out to me. The reason I set it up that was is because when I looked at the example in the readme file. It showed the return( #=> #<School:0x007fcaf4957830 @hours_in_school_day=7, @start_time="9:00", @student_names=[]>)

So I thought, I would follow how the return had it. But I do understand the specification now.